prov/efa/src/rdm/efa_rdm_ep.h

```diff
@@ -187,6 +191,9 @@ struct efa_rdm_ep {
 	struct dlist_entry entry;
 	/* the count of opes queued before handshake is made with their peers */
 	size_t ope_queued_before_handshake_cnt;
+
+	struct ofi_bufpool *peer_map_entry_pool; 	/* bufpool to hold fi_addr->efa_rdm_peer key-value pairs */
+	struct efa_rdm_peer_map peer_map; 		/* Hashmap to find efa_rdm_peer given fi_addr */
```

Comment:
The name peer_map feels a little generic.

Reply:
What would you prefer instead? `fi_add_to_peer_map`?

Reply:
Yeah that's better